ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -57,11 +57,23 @@ open class TwoLineTitleView: UIView {
         case dark
     }
 
+    @objc(MSFTwoLineTitleViewAlignment)
+    public enum Alignment: Int {
+        case center
+        case leading
+    }
+
     @objc(MSFTwoLineTitleViewInteractivePart)
     public enum InteractivePart: Int {
-        case none
-        case title
-        case subtitle
+        case none = 0
```

Comment:
Nit: I believe you can remove the ints (= 0, = 1). By default, objc enums will use ints.

Reply:
I want to be explicit here because this is supposed to be a bit mask. (See Saad's comment [here](https://github.com/microsoft/fluentui-apple/pull/1527/files/ecc87d8879573eb4403e3447a21d083eadf8a8d7#r1091178750).)

Reply:
no i think the bit operation is worse for readability. but if we want to use bit operation we should follow more the style of how apple does its like uiaccessibilitytraits. That way we can still use insert/remove bit operator

Reply:
`UIAccessibilityTraits` uses `OptionSet`, which doesn't play nicely in Objective C. This might be doable by defining the enum in Objective C using `NS_OPTIONS`, although this leaves the open question of whether the generated Swift version has the desired name or not.

Reply:
i am confused.  we can access uiaaccessbilitytraits in both objc and swift currently.

Reply:
`UIAccessibilityTraits` was originally written in Objective C and has been made available in Swift. We're trying to do the opposite; that is, taking an enum written in Swift and making it available in Objective C.